site/src/pages/docs/client-timeouts.mdx

```diff
@@ -20,6 +20,39 @@ HelloService.Iface client =
            .build(HelloService.Iface.class);
 ```
 
+## Using <type://RequestOptions>
```

Comment:
Ditto - How about splitting this into two sections, one for using preparation and the other for using options?